apps/passport-client/components/screens/LoginScreens/CreatePasswordScreen.tsx

```diff
@@ -120,15 +120,11 @@ export function CreatePasswordScreen() {
           <H2>Choose a Password</H2>
           <Spacer h={24} />
           This password will be used to generate an encryption key that secures
-          your data. Save your password somewhere you'll be able to find it
-          later. If you skip this now, you will be asked to set a password on
-          adding your first PCD.
+          your data. Save your password somewhere you'll be able to find later.
         </TextCenter>
         <Spacer h={24} />
 
         <CenterColumn>
-          <BigInput value={email} disabled={true} />
```

Comment:
I think having the email is pretty good, so we should figure out a way to add it back later imo.